ArduPlane/tiltrotor.cpp

```diff
@@ -82,6 +82,15 @@ const AP_Param::GroupInfo Tiltrotor::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("WING_FLAP", 10, Tiltrotor, flap_angle_deg, 0),
 
+    // @Param: RATE_FWD
+    // @DisplayName: Transition tilt completion rate.
+    // @Description: This is the angular rate that rotors will be tilted from Q_TILT_MAX to the fixed wing flight tilt angle when doing a transition from VTOL to fixed wing flight. If the maximum throttle of tilting motors starts to saturate, then the rotors will tilt back to Q_TILT_MAX at Q_RATE_UP deg/sec until the motors stop saturating.  
+    // @Units: deg/s
+    // @Increment: 1
+    // @Range: 5 45
+    // @User: Standard
+    AP_GROUPINFO("RATE_FWD", 11, Tiltrotor, max_rate_down_transition_dps, 15),
```

Comment:
We should probably have a think if we should enable by default or not. A half way might be for a value of 0 to use the existing tilt rate.

Reply:
i think a default of zero would be good, where zero means using the current rate down

Reply:
Maybe. We will see if that is too fast when you do your testing.

Reply:
The reasoning behind not using the current Q_TILT_RATE_DN value is that the tilt past Q_TILT_LIM should be slowed as a given change in rotor tilt in that region gives a larger change in vertical force component so a slower rate is required to prevent a loss of pitch control. I have implemented the suggestion to fall use Q_TILT_RATE_DN if Q_TILT_RATE_FWD is 0, but have left the default value at 15 until evaluated.